ext/ffi/config.m4

```diff
@@ -8,6 +8,16 @@ PHP_ARG_WITH([ffi],
 if test "$PHP_FFI" != "no"; then
   PKG_CHECK_MODULES([FFI], [libffi])
 
+  AC_MSG_CHECKING([libffi version])
+  m4_define([PHP_LIBFFI_MIN_VERSION], [3.2.1])
+  libffi_version=`$PKG_CONFIG libffi --modversion`
+  if `$PKG_CONFIG libffi --atleast-version=PHP_LIBFFI_MIN_VERSION`; then
+    AC_MSG_RESULT([$libffi_version (ok)])
+  else
+    AC_MSG_RESULT([$libffi_version])
+    AC_MSG_ERROR([libffi is too old: version ]PHP_LIBFFI_MIN_VERSION[ or greater required])
+  fi
```

Comment:
Any reason for the explicit check instead of `libffi >= 3.2.1` in PKG_CHECK_MODULES? That's what we usually do.

Reply:
That would work also but would need to be changed to integer (major*10000+minor*1000+patch...) and comparing libffi variable with string will error out with something like "integer expression expected"... Or something else here? Maybe some existing example?

Reply:
Ah, that is actually already implemented by the PKG_CHECK_MODULES macro... Ok, I understand. Let met check.